gcassert_test.go

```diff
@@ -1,6 +1,8 @@
 package gcassert
 
 import (
+	"bytes"
+	"github.com/stretchr/testify/require"
```

Comment:
nit: the order seems a bit off (`require` should go into the second group below).

Reply:
Fixed.

Reply:
Still seems off.

Reply:
Sorry, fixed.